src/components/Download/Download.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from "react";
+import "./Download.css";
+import { Table, NavLink, Input, Row, Col } from "reactstrap";
+import { IconContext } from "react-icons";
+import { FaDownload } from "react-icons/fa";
+
+export default () => {
+  const [Platform, setPlatform] = useState("");
+
+  useEffect(() => {
+    const os = window.navigator.platform;
+    setPlatform(os);
+  }, []);
+
+  const download = async () => {
+    const url =
+      "https://api.github.com/repos/MovingBlocks/TerasologyLauncher/releases/latest";
+    const response = await fetch(url);
```

Comment:
can we make use of graphQL for fetching ? 
@majordwarf  @ryuk156

Reply:
I don't think it's worth using GraphQL because to convert REST API to graphQL you'd need to create a custom schema, which is viable when you are creating multiple or nested calls to the API.
@pollend Any input?

Reply:
might want to consider the long terms plans of github in regards to graphql:  https://github.blog/2016-09-14-the-github-graphql-api/ . I don't have any background knowledge of the current state of the codebase for this website but Its not too difficult to map an internal method to another with some overhead. you can migrate things over as things progress  and kind of do something similar to the JOML stuff in the core engine.